IanGRap/Squish
# fixed the bug where blocks would get offset

Goals:
- Fix the bug where blocks would get slightly misaligned from integer x values
- Clean up the code inside BlockMovement.cs a little 

Important Files Changed:
- BlockMovement.cs

How To Test:
- Play the sprint1 demo scene and push lots of blocks around. At the end, look at the x position of all blocks. Make sure all of them are an integer value
- Test the block perfomance test (allows you to control Block (21) with WASD. Monitor Block (21)'s x position. Ensure that after every push, the x position is an integer value.
